### ISSUE TITLE
Synthtext pre processing and table 2 accuracies

### ISSUE THREAD

**Issue by crazysal** [tonghe90/textspotter#43]
Hi, 

Please can you tell the steps taken for pre-procesing synthtext labels ?? 

your model uses fixed max length of 25 but synthtext dataset has boxes with labels length(number of characters per box in ground truth)>=35

Also how did you get the accuracies mentioned in Table 2 ?
is that after all steps of training ? It says accuracy on icdar dataset but also says groundtruth used.
or is it after training on Synthtext and then fine tune of 80k iteration on Icdar ie after step 2 of training ??

**Comment by crazysal** [comment]
Training only on Synthtext with fixed trained east base(benchmark matched ) and groundtruth bounding boxes from Synthtext  gives recognition accuracy plateaued at 20%

**Comment by crazysal** [comment]
Update : 
Using pre-trained glove embedding (https://github.com/minimaxir/char-embeddings) I am getting 56.7% accuracy on SynthText now.. 

Any comments on results on SynthText would be appreciated.
